src/cli/commands/version.js

```diff
@@ -118,9 +118,7 @@ export async function setVersion(
   // check if committing the new version to git is overriden
   if (!flags.gitTagVersion || !config.getOption('version-git-tag')) {
     // Don't tag the version in Git
-    return function(): Promise<void> {
-      return Promise.resolve();
-    };
+    return Promise.resolve.bind(Promise);
```

Comment:
Sure. That said using an arrow function for this turned out to be slightly slower on V8: https://jsperf.com/resolved-promise